kubernetes/annotations_and_labels.md

```diff
@@ -205,6 +206,14 @@ and should be placed inside the chart's `templates/` subdirectory and they
 define a couple of variables that can then be re-used throughout other
 templates in the chart.
 
+The team label is set using the `maintainers` field in Chart.yaml. The prefix
+`team-` identifies it and is used to set `.Values.maintainers.team`.
+
+```
+maintainers:
+- name: team-firecracker
+```
```

Comment:
Will `.Values.maintainers.team` work with that? `maintainers` seem to be an array.

Reply:
Yes maintainers is an array so  `architect helm template` would need to parse that.

Each app would belong to a team. This also allows having squads for optional apps or individuals as extra fields in maintainers.

There is an edge case with cluster-autoscaler which belongs to 2 teams depending on the provider. There is more discussion in the issue and we still need to find a solution for that.

Reply:
Why not something like .Chart.maintainers[0].name. (You’d need to use index function iirc)

Reply:
@kopiczko Thanks I like using `.Chart.maintainers[0].name` as then we don't need to change architect and can just use Helm templating.

I'll get back to this later this week.